### ISSUE TITLE
pointer generator model starts overfitting

### ISSUE THREAD

**Issue by priyanks179** [atulkum/pointer_summarizer#28]
I am trying to train pointer generator network . After training for 10k iterations it starts overfitting.
Any suggestions on why this might be happening.
Note- 1==50 iterations
![adagrad 52k](https://user-images.githubusercontent.com/40527782/54012934-8cd5bf00-419d-11e9-93d2-857563dcfc76.png)

**Comment by atulkum** [comment]
make sure you turn off the coverage loss initially.

**Comment by priyanks179** [comment]
> make sure you turn off the coverage loss initially.

bro i am not using coverage loss at all . And model that i am using is exactly same as yours . Just the thing is that i am fixing max_enc_steps to 200 and max_dec_steps to 30 .

**Comment by priyanks179** [comment]
sir pls help coz i am trying to solve this problem for a month and i am not able to figure it out .

**Comment by priyanks179** [comment]
Sir the link to repo is below:-
https://github.com/priyanks179/pointer-gen-network

**Comment by A-Rain** [comment]
@atulkum Hello sir, I'm confuse why we should turn off coverage loss initially

**Comment by atulkum** [comment]
If you turn on coverage initially the training won't converge. 
Intuitively think about coverage as one more constraint on optimization which will restrict the space you can explore. 
That is the reason first we explore the loss space without the restriction and once it converge a bit we apply the coverage.
